requirements.txt

```diff
@@ -1,3 +1,3 @@
 requests
 pika==0.12.0
--e "git+https://github.com/HumanCellAtlas/ingest-client.git@40f2384#egg=hca_ingest"
+-e git+https://github.com/ebi-ait/ingest-client.git@7fd3c25#egg=hca_ingest
```

Comment:
[289a333](https://github.com/ebi-ait/ingest-client/commit/289a33379dcc3c036b6e755d49988fded5b99073) is the latest

Reply:
will do! thanks!